src/ripple/app/tx/impl/Transactor.cpp

```diff
@@ -40,25 +40,32 @@ namespace ripple {
 NotTEC
 preflight0(PreflightContext const& ctx)
 {
-    uint32_t const nodeNID = ctx.app.config().NETWORK_ID;
-    std::optional<uint32_t> const txNID = ctx.tx[~sfNetworkID];
-
-    if (nodeNID <= 1024)
+    if (isPseudoTx(ctx.tx))
```

Comment:
It's better not to have the empty block and check `if (!isPseudoTx(ctx.tx))`

Reply:
I'm not having much luck with testing this fix. I run a local network of 5 nodes. I added this to config:
>      [network_id]
>      1026
>      [amendment_majority_time]
>      16 minutes
I ran this to every node:
>              {
>                  "method": "feature",
>                  "params": [
>                    {
>                      "feature": "AMM",
>                      "vetoed": false
>                    }
>                  ]
>                }
This is the response:
>       {'count': 1,
>         'enabled': False,
>         'name': 'AMM',
>         'supported': True,
>         'threshold': 3,
>         'validations': 5,
>         'vetoed': False}

Reply:
I'll re-test with the suggested wait time to double check.

Reply:
I waited 45 minutes - the same result.